static/js/components/LearnerSearch.js

```diff
@@ -126,6 +129,14 @@ export default class LearnerSearch extends SearchkitComponent {
   countryNameTranslations: Object = makeCountryNameTranslations();
   degreeTranslations: Object = makeDegreeTranslations();
 
+  constructor(props: Object) {
+    super(props);
+    this.WrappedLearnerResult = wrapWithProps(
+      {openLearnerEmailComposer: this.props.openLearnerEmailComposer},
```

Comment:
I did this because there was a visually-noticeable performance penalty if the component wrapping is performed in, say, the `render` method. It's also important to note that this method can't change. It's handed down from the email HOC, which is only executed once on the container component, and the function is not part of the state, so it can't change via an action/reducer.

I agree that this is kind of a code smell and it isn't by-the-book React, but I don't quite know how else to handle this without repeatedly re-wrapping this component unnecessarily.